packages/example/package.json

```diff
@@ -17,6 +17,7 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@navigation-ex/native": "^0.0.1",
```

Comment:
I think this will create a symlink, right? if just add it to the eslint config in root